src/react-components/src/__tests__/components/SearchResultItem-test.tsx

```diff
@@ -0,0 +1,51 @@
+import { expect } from "chai";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+
+import SearchResultItem from "../../components/02-molecules/Cards/SearchResultItem";
+
+describe("SearchResultItem", () => {
+  const validHeading = <a href="hello">headingText</a>;
```

Comment:
From your earlier comment, it does make sense to have the logic and this test in the `Heading` component.